Yoast/javascript
# add Dutch verb exceptions

## Summary

This PR can be summarized in the following changelog entry:

* Creates the forms of Dutch strong verbs.
All the verb forms from the exceptions list will be generated. 
However, inseparable compound verb would not be generated correctly. 
e.g. expected: behangen, received: begehangen.
This problem will be separately addressed in different issue ( generating past participle issue).

## Test instructions

Add more words from Dutch strong verbs exception list (strongVerbStems). The word can be taken from which ever stem listed in the list, e.g. present stem, past stem, or participle stem. The following should be tested:
* when the present stem ends in "t", additional suffix -t should not be added to create 3rd present singular form.
* stem modification should be implemented when necessary before attaching suffix -en and -end. However, stem modification should not be implemented on past stem of noConsonantDoublingStrongVerbsEnEnding before attaching -en.
* compound verb forms of strong verbs. The compound prefix can be found under compoundVerbsPrefixes -> separable. When the stem is preceded by one of the separable compound prefixes, the prefix should be attached back in front of all the created forms.
* whether strong verbs with irregular past form get the correct forms. The stem list can be found under irregularStrongVerbs.
* whether strong verbs with both regular and irregular past forms get the correct forms. The stem list can be found under bothRegularAndIrregularStrongVerbs.

## Quality assurance


Fixes #370
